.changeset/short-experts-flow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris': patch
+---
+
+Update Text componant to work with Labels
```

Comment:
Since this is going into the `typography-foundations-text-component` which already has a changeset file, you can delete this file since the functionality being removed hasn't been shipped in a release.

Reply:
Yep add the `skip-changelog` label to pass the check